src/lib/interactionManager/interaction.ts

```diff
@@ -202,19 +204,16 @@ export class Interaction {
           headers: { 'Content-Type': 'application/json' },
         })
 
-        try {
-          return JolocomLib.parse.interactionToken.fromJWT<T>(
-            (await response.json()).token,
-          )
-        } catch {
-          return
-        }
+        return JolocomLib.parse.interactionToken.fromJWT<T>(
+          (await response.json()).token)
+
```

Comment:
This change breaks `CredentialShare` and `Auth` flows, since the request doesn't return a token. Didn't notice until now. Maybe would be better to extract the parsing outside the `send` method and apply it only to `CredentialReceive`

Reply:
Good catch, will try the suggestion now.